afs/afs-ws/afs-ws-client-utils/src/main/java/com/powsybl/afs/ws/client/utils/RemoteServiceConfig.java

```diff
@@ -27,6 +27,14 @@
 
     private boolean secure;
 
+    private boolean autoreconnect;
+
+    private int reconnectionInitialInterval;
+
+    private int reconnectionIntervalMutiplier;
+
+    private int reconnectionMaxInterval;
```

Comment:
It should be easier to configure if we have a « maxRetry » or « MaxAttemps » instead of a « MaxInterval ».

Reply:
Added intervalMax. see the link
 docs/configuration/modules/remote-service.md